src/main/java/jackbean/command/Ui.java

```diff
@@ -0,0 +1,44 @@
+package jackbean.command;
+
+/**
+ * Represents the user interface of the JackBean chatbot.
+ */
+public class Ui {
+    private static final String horizontalLine = "____________________________________________________________";
```

Comment:
Should the name horizontalLine be in capital letters since it is a constant?

Reply:
Ooh, thanks for catching that JianZhi!